_beats/dev-tools/common.bash

```diff
@@ -34,18 +34,18 @@ get_go_version() {
   fi
 }
 
-# install_gimme
-# Install gimme to HOME/bin.
-install_gimme() {
-  # Install gimme
-  if [ ! -f "${HOME}/bin/gimme" ]; then
-    mkdir -p ${HOME}/bin
-    curl -sL -o ${HOME}/bin/gimme https://raw.githubusercontent.com/travis-ci/gimme/v1.1.0/gimme
-    chmod +x ${HOME}/bin/gimme
+# install_gvm
```

Comment:
I'm not sure if this will break the release entry-point ... just to let you know I went through some pain when I removed the gimme support in Beats and somehow it was not used in the beats-ci but the release process used it

Reply:
I don't like the idea of renaming the function back to `install_gimme` 😄. I'll check infra's jobs to do that. Besides that, master branch is not using gvm nor `withGoEnv`, but OTOH I'd like to add consistency across our projects

Reply:
Cannot find any occurrence of `install_gimme` at infra's repo. I'd say the entrypoints are related to build command than to shell functions.